jstreegrid.js

```diff
@@ -966,6 +969,10 @@
 						} else {
 							last.removeClass(SEARCHCLASS);
 						}
+						// TODO this does not work for the tree column yet
+						if (i !== treecol && col.ellipsis !== false) {
```

Comment:
````javascript
if (I !== treecol && (col.ellipsis === true || (col.ellipsis === undefined && this.settings.themes.ellipsis === true))) {
````

(I might have the above wrong in terms of retrieving the setting, but the idea)